Neos.Neos/Resources/Private/Fusion/Prototypes/Shortcut.fusion

```diff
@@ -1,15 +1,83 @@
 # Neos.Neos.Shortcut is given a representation for editing purposes
 #
-prototype(Neos.Neos:Shortcut) < prototype(Neos.Fusion:Template) {
-  templatePath = 'resource://Neos.Neos/Private/Templates/FusionObjects/Shortcut.html'
-
+prototype(Neos.Neos:Shortcut) < prototype(Neos.Fusion:Component) {
   targetMode = ${q(node).property('targetMode')}
   firstChildNode = ${q(node).children('[instanceof Neos.Neos:Document]').get(0)}
   target = ${q(node).property('target')}
   targetConverted = ${Neos.Link.hasSupportedScheme(this.target) ? Neos.Link.convertUriToObject(this.target, node) : null}
   targetSchema = ${Neos.Link.getScheme(this.target)}
 
-  node = ${node}
+  i18n = ${I18n.id(null).package('Neos.Neos').source('Main').locale(Neos.Backend.interfaceLanguage())}
+
+  renderer = Neos.Fusion:Match {
+    @subject = ${props.targetMode}
+
+    selectedTarget = Neos.Fusion:Join {
+      @glue = '<br/>'
+      mainMessage = ${props.i18n.id('shortcut.toSpecificTarget').translate()}
+      targetSelected = Neos.Fusion:Match {
+        @if.isTarget = ${props.target}
+
+        @subject = ${props.targetSchema}
+
+        node = Neos.Fusion:Value {
+          targetUriTag = Neos.Neos:NodeLink {
+            node = ${props.targetConverted}
+          }
+          value = ${props.i18n.id('shortcut.clickToContinueToPage').arguments([this.targetUriTag]).translate()}
+        }
+
+        asset = Neos.Fusion:Value {
+          targetUriTag = afx`
+            <a target='_blank'>
+              <Neos.Fusion:ResourceUri @path='attributes.href' resource={props.targetConverted.resource} />
+              {props.targetConverted.label}
+            </a>
+          `
```

Comment:
I would prefer pure Fusion here…

Reply:
is adjusted ;)

Funnily i had it that way first https://github.com/neos/neos-development-collection/pull/3372#discussion_r663644088